### DISCUSSION TITLE
Add support for ECS,EKS and RDS

### DISCUSSION THREAD

**Discussion by sammcj** [opengovern/opensecurity#40] [Ideas]
**Is your feature request related to a problem? Please describe.**

These days most clients I work with don't run any EC2 instances anymore, everything is containerised.

Current containerised services and SaaS databases don't show up in Kaytu.

**Describe the solution you'd like**

It would be great if this tool could add support for ECS,EKS and RDS.

**Describe alternatives you've considered**

N/A

**Additional context**

N/A

**Comment by acx1729** [comment]
Hi @sammcj, thanks for the input. We are working on RDS and we should be able to show a preview this weekend. We will raise tasks on EKS.

**Reply by sammcj** [reply]
That's fantastic news! Thanks :)

**Reply by acx1729** [reply]
Hi @sammcj, The RDS feature is available in the latest release. We are testing RDS on workload. Would love to get your feedback.

**Reply by sammcj** [reply]
Not sure if I'm imagining this but wasn't the CLI previously interactive?

```
kaytu
Usage:
  kaytu [flags]
  kaytu [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  login
  logout
  optimize
  plugin
  version

Flags:
  -h, --help   help for kaytu

Use "kaytu [command] --help" for more information about a command.
```

I would run optimise but it's name suggests it might make changes?

**Reply by salehkhazaei** [reply]
@sammcj we only do read-only actions, no change will be applied by kaytu on any command. the name might be misleading but it wont effect your resources.

**Reply by acx1729** [reply]
@sammcj, would "check" or "scan" be more accurate?

**Reply by sammcj** [reply]
Ah, yes that would make me (and I'm assuming others) feel more safe running the tool against their infra!

I think 'check' feels the safest.
